konfig

```diff
@@ -123,13 +126,15 @@ export_contexts() {
     fi
 
     for x in "${ctxs[@]}"; do
-      tmpcfg=$(mktemp konfig_XXX)
+      tmpcfg=$(mktemp konfig_XXXXXX)
       TMPFILES+=( "$tmpcfg" )
       tmpcfgs+=( "$tmpcfg" )
       if [[ "${#cfgs[@]}" -eq 0 ]]; then
         $KUBECTL config view --flatten --minify --context="$x" > "$tmpcfg"
       else
+        IFS=$':\n\t'
         KUBECONFIG="${cfgs[*]}" $KUBECTL config view --flatten --minify --context="$x" > "$tmpcfg"
+        IFS=$' \n\t'
```

Comment:
Because here you set IFS in the context of this function, the scope fill automatically be reset when the function completes. It is thus not necessary to do it manually.

Reply:
Are you sure? `IFS` is a global variable, so any applied changes should persist when the function exits.